### ISSUE TITLE
In which scenarios are certificates shown with a grey background?

### ISSUE THREAD

**Issue by Ein-Tim** [corona-warn-app/cwa-app-ios#3421]
## Avoid duplicates
* [X] Question is not already answered in the [FAQ](https://www.coronawarn.app/en/faq/)
* [X] Question is specific for iOS only, for general issues / questions that apply to iOS and Android please raise them in the [documentation repository](https://github.com/corona-warn-app/cwa-documentation)
* [X] Question has not already been asked in another issue

## Your Question
In which scenarios are certificates shown with a grey background?
My thoughts:
1. First vaccination certificate is shown with a grey background **only if** certificate from second vaccination is shown too
2. The certificates is not the "currently used certificate" 
3. The certificates are invalid

Can you confirm this?

## Additional information
I had a very long discussion about this with a Twitter user: https://twitter.com/frauprince/status/1430491429638377477?s=20

**Comment by Ein-Tim** [comment]
@dsarkar I guess this snowed under?

**Comment by dsarkar** [comment]
@Ein-Tim Indeed... stand by.

**Comment by dsarkar** [comment]
@Ein-Tim I got feedback from the UX team: 

"The currently used certificate, if it is technically valid, is displayed in blue. All others are displayed in gray.
If the currently used certificate is invalid, it will be displayed in gray with a warning icon (and in most cases with a grayed out QR code).
 
The blue is used to highlight the currently used technically valid certificate. Since this is probably the most "valuable" certificate for the user at the moment. Everything else is gray. We have refrained from further color coding, since we cannot say anything about the "fachliche" validity, except when checking the travel rules."
